datalad/downloaders/http.py

```diff
@@ -517,9 +517,20 @@ class HTTPDownloader(BaseDownloader):
     """
 
     @borrowkwargs(BaseDownloader)
-    def __init__(self, headers={}, **kwargs):
+    def __init__(self, headers=None, **kwargs):
+        """
+
+        Parameters
+        ----------
+        headers: dict, optional
+          Header fields to be provided to the session. Unless User-Agent provided, a custom
+          one, available in `DEFAULT_USER_AGENT` constant of this module will be used.
+        """
         super(HTTPDownloader, self).__init__(**kwargs)
         self._session = None
+        headers = headers and headers.copy() or {}
```

Comment:
explicit better than obscure, so let's make it
```suggestion
        headers = headers.copy() if headers else {}
```

Reply:
FTR: I would be fine with both patterns